frontend/web/components/GitHubResourcesSelect.tsx

```diff
@@ -153,43 +67,74 @@ const GitHubResourcesSelect: FC<GitHubResourcesSelectType> = ({
   }, [lastSavedResource, selectedOption])
 
   return (
-    <div>
-      <Select
-        filterOption={(options: any[]) => {
-          return options
-        }}
-        value={selectedOption}
-        size='select-md'
-        placeholder={'Select Your Resource'}
-        onChange={(v: GitHubResourcesValueType) => {
-          setSelectedOption(v)
-          onChange(v?.value)
-        }}
-        isClearable={true}
-        options={githubResources?.map((i: GithubResource) => {
-          return {
-            label: `${i.title} #${i.number}`,
-            value: i,
-          }
-        })}
-        noOptionsMessage={() =>
-          isLoading
-            ? 'Loading...'
-            : isFetching
-            ? 'Refreshing data ...'
-            : 'No Results found'
-        }
-        onInputChange={(e: any) => {
-          setSearchText(e)
-          searchItems(Utils.safeParseEventValue(e))
-        }}
-        components={{
-          Control: CustomControl,
-          MenuList,
-        }}
-        data={{ searchText }}
-      />
-    </div>
+    <>
+      <div className='d-flex gap-2 mb-2'>
+        <MyRepositoriesSelect
+          githubId={githubId}
+          orgId={orgId}
+          value={repo}
+          onChange={setRepo}
+        />
+        <div style={{ width: 200 }}>
+          <Select
+            autoSelect
+            className='w-100'
+            size='select-md'
+            placeholder={'Select Type'}
+            value={githubTypes.find((v) => v.resourceType === resourceType)}
+            onChange={(v: { resourceType: string }) =>
+              setResourceType(v.resourceType)
+            }
+            options={githubTypes.map((e) => {
+              return {
+                label: e.label,
+                resourceType: e.resourceType,
+                value: e.id,
+              }
+            })}
+          />
+        </div>
+      </div>
+      {!!repoName && !!repoOwner && (
+        <div>
+          <label className='cols-sm-2 control-label'>
+            Link new Issue / Pull Request
+          </label>
+          <Select
```

Comment:
An issue that is already linked is still appearing in the list, which will cause an error with the backend when trying to select it again.

<img width="576" alt="Screenshot 2024-08-15 at 10 01 56 AM" src="https://github.com/user-attachments/assets/f6229bce-adca-4a34-9fd5-b71abe41fb79">

<img width="578" alt="Screenshot 2024-08-15 at 10 02 03 AM" src="https://github.com/user-attachments/assets/87a68ab4-266b-4935-8035-6cfcb8498128">

Reply:
Ah didn't spot this in the original provider. Sorted